pkg/controller/plan/kubevirt.go

```diff
@@ -857,6 +859,48 @@ func (r *KubeVirt) GetGuestConversionPod(vm *plan.VMStatus) (pod *core.Pod, err
 	return
 }
 
+func (r *KubeVirt) GetVirtV2VDiskConfig(vm *plan.VMStatus, pod *core.Pod, step *plan.Step) (err error) {
+
+	if pod.Status.PodIP == "" {
+		//we need the IP for fetching the fimware info.
```

Comment:
did you consider creating a service instead?
and there's a typo:
```suggestion
		//we need the IP for fetching the firmware info.
```

Reply:
I did initially, and then I saw that the monitor part is using the same mechanism and its simpler this way, not sure if its worth the addition of the service creation and then adding its query just for getting a single value.

Reply:
that's a fair point - creating a service would complicate the flow and we already do it the same way for the progress
but then we can get the OVF after the 'conversion' step is completed - at that point the pod is definitely running and has the OVF ready

Reply:
That's another point, since we currently depend on pod completion to continue to the next step in the migration process, leaving it running to retrieve additional information later would require altering this flow as well. So I think at this point its better to leave it as it is and we can think on this improvement in a later stage.